rules/precompiled_apple_resource_bundle.bzl

```diff
@@ -121,7 +121,7 @@ def _precompiled_apple_resource_bundle_impl(ctx):
     control_files = []
     input_files = []
     output_files = []
-    output_bundle_dir = ctx.actions.declare_directory(paths.join(ctx.attr.name, bundle_name + ".bundle"))
+    output_bundle_dir = ctx.actions.declare_directory(paths.join(ctx.attr.name, bundle_name + ctx.attr.bundle_extension))
```

Comment:
This wasn't a functional change but seemed appropriate if the attribute exists already